ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/port-channel-interfaces.j2

```diff
@@ -80,6 +80,21 @@ interface {{ port_channel_interface }}
 {%         if port_channel_interfaces[port_channel_interface].rt is arista.avd.defined %}
       route-target import {{ port_channel_interfaces[port_channel_interface].rt }}
 {%         endif %}
+{%         if port_channel_interfaces[port_channel_interface].redundancy is arista.avd.defined %}
+      redundancy {{ port_channel_interfaces[port_channel_interface].redundancy }}
+{%         endif %}
+{%         if port_channel_interfaces[port_channel_interface].designated_forwarder_election is arista.avd.defined and port_channel_interfaces[port_channel_interface].designated_forwarder_election.algorithm is arista.avd.defined %}
+{%             set designated_forwarder_cli = "designated-forwarder election algorithm " %}
```

Comment:
Avoid leaving an extra space at the end here. Instead add the space before the variables when you add each under the `if`s below. This way it will not render an empty space in the case where the none of the conditions are true. It will also make it simpler to add new stuff later.